### PR TITLE
Working with 1.17.1

### PR DESCRIPTION
**Pull request by JacobKorn**
- updated to jdk 16
- stopped specifying the garbage collector (it's been removed from java 16) in the run command.

I'm not sure if we're supposed to specify another garbage collector, but it seems to work.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+mc_data
```
Comment (ruippeixotog) [inline .gitignore]: Why is this needed?

**Review by ruippeixotog — COMMENTED** [review]
Thanks for the update @JacobKorn!

File: .gitignore (hunk 1 above)
Comment (JacobKorn) [reply .gitignore]: Just to avoid accidental commits and to stop vs code trying to index it.

But technically it's not needed.

Comment (ruippeixotog) [reply .gitignore]: But neither creating this image or running it write an `mc_data` folder unless you explicitly add it as a volume, right? That makes the scenario pretty specific to that workflow, so I'd rather leave .gitignore clean. Other than that LGTM 🙂

**Review by ruippeixotog — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (JacobKorn) [reply .gitignore]: Ah yes, I see what you mean. Makes sense :+1:

**Comment by JacobKorn** [comment]
Your explanation makes sense, I removed the .gitignore.

**Review by ruippeixotog — APPROVED** [review]
